### ISSUE TITLE
commands.Bot command not sending UI View

### ISSUE THREAD

**Issue by tonymoooon543** [Pycord-Development/pycord#209]
### Summary

Trying to get a command to send a message with a View will raise a error

### Reproduction Steps

Create a `commands.Bot` bot, add command which send a UI View.

### Minimal Reproducible Code

```python
class MyView(discord.ui.View):
    @discord.ui.button(label='A button', style=discord.ButtonStyle.primary)
    async def button_callback(self, button, interaction):
        await interaction.response.send_message('Button was pressed', ephemeral=True)

@commands.command(name='command')
async def command(ctx):
    view = MyView()
    await ctx.send("Hello World!", view=view)
```

### Expected Results

When you use the command it sends a message with a UI view

### Actual Results

Traceback (most recent call last):
  File "C:\...\Python39\lib\site-packages\disocord\ext\commands\bot.py", line 994, in invoke
    await ctx.command.invoke(ctx)
  File "C:\...\Python39\lib\site-packages\discord\ext\commands\core.py", line 894, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "C:\...\Python39\lib\site-packages\discord\ext\commands\core.py", line 176, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: InvalidArgument: view parameter must be View not <class '__main__.MyView'>  

### Intents

default 

### System Information

- Python v3.9.5-final
- discord v1.7.3-final
- aiohttp v3.6.3
- system info: Windows 10 10.0.18363

### Checklist

- [x] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

**Comment by jgayfer** [comment]
We'd need to see what `MyView` looks like to debug this. Is it subclassing `discord.ui.View`?

**Comment by tonymoooon543** [comment]
Fixed it, updating to 2.0.0 Alpha fixed the issue
